common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -455,6 +469,8 @@ private void log(final String tag,
         sLogExecutor.execute(new Runnable() {
             @Override
             public void run() {
+                final String dateTimeStamp = getUTCDateTimeAsString();
```

Comment:
We should move this up (approx. `L468`) so that the timestamp is generated on the _calling thread_ not the worker